ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -108,6 +109,12 @@
     reason="Test runs ONLY on cloud based deployed cluster"
 )
 
+on_prem_platform_required = pytest.mark.skipif(
+    config.ENV_DATA['platform'].lower() not in ON_PREM_PLATFORMS,
```

Comment:
shouldn't the RGW tests run on independent mode clusters only?

Reply:
No, we have two epics - one for converged, and one for independent.
https://issues.redhat.com/browse/KNIP-1420
https://issues.redhat.com/browse/KNIP-1421

Reply:
please ignore :)